### PR TITLE
feat: add MQTT integration to drogue-cloud-server

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: server/src/main.rs [hunk 1]
```diff
@@ -260,6 +260,11 @@ fn main() {
                         .long("--enable-mqtt-endpoint")
                         .help("enable mqtt endpoint"),
                 )
+                .arg(
+                    Arg::with_name("enable-mqtt-integration")
+                        .long("--enable-integration-endpoint")
```
Comment (lulf) [inline server/src/main.rs:265]: I think name and arg should both be '--enable-mqtt-integration' should be the same to be consistent with the other options. Also I imagine websocket would be '--enable-websocket-integration'.

File: server/src/main.rs [hunk 2]
```diff
@@ -570,6 +578,37 @@ fn main() {
...
+        if matches.is_present("enable-mqtt-integration") || matches.is_present("enable-all") {
+            log::info!("Enabling MQTT integration");
+            let bind_addr_mqtt = server.mqtt_integration.clone().into();
+            let kafka = server.kafka.clone();
+            threads.push(std::thread::spawn(move || {
+                let config = drogue_cloud_mqtt_integration::Config {
+                    health: None,
+                    oauth,
+                    disable_tls: true,
+                    cert_bundle_file: None,
```
Comment (lulf) [inline server/src/main.rs:590]: Can this use the same logic as for the http endpoint to enable TLS?

**Review by lulf — CHANGES_REQUESTED** [review]
Looks good overall, a few comments.

File: server/src/main.rs (hunk 2 above)
Comment (ctron) [reply server/src/main.rs:590]: Ah sure. I copied the logic from the MQTT endpoint. I will fix both.

**Review by lulf — APPROVED** [review]
LGTM!

**Pull request merged**